snapshots/war/2024-01-11/nuclear_threat_initiative_overview.py

```diff
@@ -0,0 +1,76 @@
+"""Script to create a snapshot of dataset.
+
+We do not extract any data from NTI directly. Instead, we visually inspect
+https://www.nti.org/countries/
+And manually create a table of countries' status on nuclear weapons from 2017 onwards, to complement the data provided
+by Philipp C. Bleek, "When Did (and Didn't) States Proliferate? Chronicling the Spread of Nuclear Weapons".
+
+"""
+
+from pathlib import Path
+
+import click
+import pandas as pd
+
+from etl.snapshot import Snapshot
+
+# Version for current snapshot dataset.
+SNAPSHOT_VERSION = Path(__file__).parent.name
+
+# Latest year for which we have assume NTI has data.
+LATEST_YEAR = 2022
```

Comment:
Should that year be 2023 now?

Reply:
Yes, Bastian has confirmed that 2023 is also fully informed by the NTI.